### ISSUE TITLE
Xlsx reader not recognize data validations that references another sheet - Again

### ISSUE THREAD

**Issue by otargetj2s** [PHPOffice/PhpSpreadsheet#2149]
This is:

```
- [x] a bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```

### What is the expected behavior?
Load the data validations that references lists of another sheet.

### What is the current behavior?
return data validations with none type. Data not loaded for external list.
This regression was reported in early 2018. Never merged.

Excel sample:
[sample.xlsx](https://github.com/PHPOffice/PhpSpreadsheet/files/6622841/sample.xlsx)

See links :
https://github.com/PHPOffice/PhpSpreadsheet/issues/388
https://github.com/PHPOffice/PhpSpreadsheet/issues/991
https://github.com/PHPOffice/PhpSpreadsheet/issues/1432

### What are the steps to reproduce?
Create a xlsx file with a column with data validation that reference a list of another sheet, then load it in your code.

Please provide a [Minimal, Complete, and Verifiable example](https://stackoverflow.com/help/mcve) of code that exhibits the issue without relying on an external Excel file or a web server:

```php
<?php

require __DIR__ . '/vendor/autoload.php';

// Create new Reader and Spreadsheet object
$reader   = \PhpOffice\PhpSpreadsheet\IOFactory::createReader('Xlsx');
$spreadsheet = $reader->load('sample.xlsx');

// add code that show the issue here...
$spreadsheet->setActiveSheetIndexByName('Feuil1');
$worksheet  = $spreadsheet->getActiveSheet();

// internal drop down list: success (type "list")
echo 'B5:DrowDownListType: ', $worksheet->getCell('B5')->getDataValidation()->getType(), "\n";
// external drop down list: failure, data not loaded (type "none" instead of "list")
echo 'B14:DrowDownListType: ', $worksheet->getCell('B14')->getDataValidation()->getType(), "\n";
```

### Which versions of PhpSpreadsheet and PHP are affected?
phpspreadsheet >= 1.6.0
php >= 7.2

**Comment by otargetj2s** [comment]
See the following link for code correction :
https://github.com/PHPOffice/PhpSpreadsheet/issues/1432#issuecomment-640028401

Can you do what's nessessary? I absolutely need it for my development.
Thanks.

**Comment by PowerKiKi** [comment]
Fixed in [1.19.0](https://github.com/PHPOffice/PhpSpreadsheet/releases/tag/1.19.0)
